lib/Conversion/ArcToLLVM/LowerArcToLLVM.cpp

```diff
@@ -440,20 +441,19 @@ struct SimGetPortOpLowering : public ModelAwarePattern<arc::SimGetPortOp> {
                            .getValue());
     ModelInfoMap &model = modelIt->second;
 
+    auto type = typeConverter->convertType(op.getValue().getType());
```

Comment:
Nit: We don't specifically restrict the type of arc model ports, right? So maybe we should check for null type here and return failure? Or at least assert?

Reply:
Good point! Fixed